port/unix/omrsysinfo.c

```diff
@@ -5999,26 +6243,27 @@ omrsysinfo_cgroup_is_system_available(struct OMRPortLibrary *portLibrary)
 
```

Comment:
if we begin with `int32_t rc = 0`, we can avoid the `rc = 0` assignments, and an `else` block. `rc` will only be set for an error. Will this suggestion work?